gitpod-io/gitpod
# Update timeouts layout in user preferences

## Description

Minor layout updates to the timeouts settings in user preferences:
1. Simplify section title 
2. Remove top margin above the text input
3. Decrease font weight on the help text

| BEFORE | AFTER |
|-|-|
| <img width="642" alt="timeouts-before" src="https://user-images.githubusercontent.com/120486/222687047-a528687b-6732-402d-bdd5-797ecd3789b7.png"> | <img width="642" alt="timeouts-after" src="https://user-images.githubusercontent.com/120486/222686990-c7ec476d-5b9f-4878-9c07-7ce4d8bac884.png"> |

## How to test

Go to `/user/preferences` and notice the different layout in the timeouts section.

## Release Notes

```release-note
NONE
```

## Documentation




## Build Options:

      Experimental feature to run the build with GitHub Actions (and not in Werft).
      leeway-target=components:all
      Run Leeway with `--dont-test`

<details>
<summary>Publish Options</summary>

</details>

<details>
<summary>Installer Options</summary>

  Add desired feature flags to the end of the line above, space separated
</details>

#### Preview Environment Options:
      If enabled this will build `install/preview`
      If enabled this will create the environment on GCE infra
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`
